metabase/metabase
# Fix the problem with unknown fields in Dashcard mapping

Related to https://github.com/metabase/metabase/issues/38225

### Description

Because of differences between MLv2 and MLv1, `/api/dashboard/:id` sends `parameter_mappings` without `base-type` attribute.

This PR implements naive approach to compare targets, proper fix must be done on BE IMO

<img width="482" alt="image" src="https://github.com/metabase/metabase/assets/125459446/e901b208-35fc-4eba-89a9-294268f27e2c">


### How to verify
- you need to have a dashboard with filters and questions with join-alias specified
- edit this dashboard, click on filter, map filter to the field with join alias
- save dashboard
- edit again, open mappings - you'll see `unknown field`

### Checklist
